### PR TITLE
Merged win32 and x11 modules into ui

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Next step to remove widgets module. 
`X11` and `Win32` modules merged with `ui`
depends on https://github.com/cogneco/ooc-kean/pull/817

### CODE REVIEW

**Comment by marcusnaslund** [comment]
#817 is merged.

**Comment by sebastianbaginski** [comment]
@marcusnaslund : thanks
@jonathanudd : can you review

File: ooc-ui.use [hunk 1]
```diff
@@ -1,4 +1,12 @@
 Name: ooc UI
 Description: User interface stuff
 SourcePath: source/ui
+SourcePath: source/ui/x11
+SourcePath: source/ui/win32
```
Comment (jonathanudd) [inline ooc-ui.use]: I don't think you need three `SourcePath`. `SourcePath: source/ui` would be enough.

**Comment by jonathanudd** [comment]
Apart from the three `SourcePath` lines I think it looks good.

File: ooc-ui.use (hunk 1 above)
Comment (sebastianbaginski) [inline ooc-ui.use]: You're right, it works. Updated.

File: ooc-ui.use [hunk 2]
```diff
@@ -2,3 +2,9 @@ Name: ooc UI
 Description: User interface stuff
```
Comment (marcusnaslund) [inline ooc-ui.use]: Terrible, terrible, but not this PR. :)

**Pull request merged**